service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -5,7 +5,7 @@ include::modules/ossm-document-attributes.adoc[]
 
 toc::[]
 
-To access the most current features of {ProductName}, upgrade to the current version, 2.0. Upgrading from version 1.1 to 2.0 requires manual steps that migrate your workloads and apps to a new instance of {ProductName} running the new version.
+Upgrading from version 1.1 to 2.0 requires manual steps that migrate your workloads and apps to a new instance of {ProductName} running the new version.
```

Comment:
Just checking that you really want "apps" in the sentence instead of "applications."

Reply:
No, I did not mean to do that. Thanks for pointing it out. :)